client/modules/IDE/components/Preferences.jsx

```diff
@@ -32,6 +32,7 @@ class Preferences extends React.Component {
       value = 8;
     }
     this.props.setFontSize(value);
+    event.target.value = value;
```

Comment:
what if, rather than doing this, the value is stored on the component state, and then onSubmit/onBlur it gets propagated to the redux state?